src/custom/hooks/web3.ts

```diff
@@ -0,0 +1,128 @@
+import { Web3Provider } from '@ethersproject/providers'
+import { useWeb3React as useWeb3ReactCore } from '@web3-react/core'
+import { Web3ReactContextInterface } from '@web3-react/core/dist/types'
+import { useEffect, useState, useCallback } from 'react'
+import { isMobile } from 'react-device-detect'
+import { injected, walletconnect, getProviderType, WalletProvider } from 'connectors'
+import { NetworkContextName } from 'constants/misc'
+
+export function useActiveWeb3React(): Web3ReactContextInterface<Web3Provider> {
```

Comment:
You don't seem to be using this hook. Is it intentional?

Reply:
Yes, as I mentioned above it looks to me that we either include this and hook in the mod file or we have to update import path in a lot of places to use original file because the compiler will first look into this mod file.

Reply:
Does it work if you re-export the hook instead of re-declaring it?

Reply:
can just  re-export it as @alfetopito says